docs/book/src/examples/subcurrency.md

```diff
@@ -1,53 +1,53 @@
-## Subcurrency
+## subcurrency
```

Comment:
The changes to this file are purely to make it all lowercase, AFAICT.  Is this deliberate?

Reply:
Oh damn it... must've changed everything to lowercase in vim by mistake.. let me fix.